apps/builder/app/builder/features/style-panel/style-source/style-source.tsx

```diff
@@ -151,7 +151,7 @@ const EditableText = ({
 const StyleSourceContainer = styled(Box, {
   display: "inline-flex",
   borderRadius: theme.borderRadius[3],
-  minWidth: theme.spacing[13],
+  minWidth: theme.spacing[15],
```

Comment:
this seems big enough
```suggestion
  minWidth: theme.spacing[14],
```